application/src/main/java/org/opentripplanner/model/plan/paging/cursor/PageCursorInput.java

```diff
@@ -1,29 +1,44 @@
 package org.opentripplanner.model.plan.paging.cursor;
 
 import java.time.Instant;
+import org.opentripplanner.framework.model.Cost;
 import org.opentripplanner.model.plan.ItinerarySortKey;
 
 /**
- * This class holds information needed to create the next/previous page cursors when there were
- * itineraries removed due to cropping the list of itineraries using the numItineraries parameter.
+ * This class holds information needed to create the next/previous page cursors either when there were
+ * itineraries removed due to cropping the list of itineraries using the numItineraries parameter or
+ * when the {@link org.opentripplanner.routing.algorithm.filterchain.filters.transit.RemoveTransitIfStreetOnlyIsBetter}
+ * filter is used.
  * <p>
  * The Instant fields come from the sets of itineraries that were removed and the ones that were
  * kept as a result of using the numItineraries parameter.
  */
 public interface PageCursorInput {
   /**
-   * The earliest-removed-departure defines the start of the search-window following the
+   * The earliest removed departure defines the start of the search window following the
    * current window. To include this removed itinerary (and all other removed itineraries)
-   * in the next-page search the search windows must overlap.
+   * in the next page search the search windows must overlap.
    */
   Instant earliestRemovedDeparture();
+
+  /**
+   * The latest removed departure together with a search window duration is used to
+   * calculate the start of the search window preceding the current window.
+   */
   Instant latestRemovedDeparture();
```

Comment:
Document whether this is the arrival or departure time of the latest removed departure.

Reply:
I think because the name has departure in it, it is clear enough, is that ok? The word "arrival" (or an abbreviation) is usually included in variables that deal with arrival times, for example in `PageCursorFactory` the variable `lat` (latest arrival time)

Reply:
latestRemovedDepartureTime would be slightly more explicit as departure is sometimes a synonym to trip and a trip can be identified by an arrival time.

Reply:
I think there was a comment here from Thomas that agreed with me, but has now disappeared. Is keeping the name the same ok for you @optionsome